### PR TITLE
Swiftlint

### PR DESCRIPTION
**Pull request by davideme**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,15 @@
 language: objective-c
 osx_image: xcode7.1
 
+before_install:
+  - brew update
+  - brew install swiftlint
```
Comment (pedrovgs) [inline .travis.yml]: Shouldn't this be part of the install process?

**Comment by pedrovgs** [comment]
:+1: thanks @davideme

File: .travis.yml (hunk 1 above)
Comment (davideme) [inline .travis.yml]: Maybe, I'm using the travis doc to setup this.
http://docs.travis-ci.com/user/osx-ci-environment/#Homebrew

Comment (davideme) [inline .travis.yml]: I read the Travis doc, and it seems that install should be used to install the project not the tools.
Like installing dependencies.

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
+  - brew update
+  - brew install swiftlint 
+
 install:
-    - gem install xcpretty
-    - gem install cocoapods
-    - pod install
+  - gem install xcpretty
+  - gem install cocoapods
+  - pod install
```
Comment (davideme) [inline .travis.yml]: gem install xcpretty
gem install cocoapods

Should be moved in the before_install and only keep the pod install

Comment (pedrovgs) [inline .travis.yml]: Can you move that part of the script to the before_install before to merge this PR?

**Pull request merged**